### PR TITLE
fix(audit-service): enable audit service to archive zero entries without error 

### LINKED ISSUE #1762: audit service giving error when no record is selected
**Describe the bug**
Audit service giving error when no record is selected to be archived. It is throwing an internal server error currently

**Expected behavior**
Output should be 
{
  "message": "No entry selected",
  "numberOfEntriesArchived": 0,
  "key": ""
}

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by arpit1503khanna**
## Description
Enable audit service to archive zero entries without error

Fixes #1762 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ $ npm install -g @sourceloop/cli
 $ sl COMMAND
 running command...
 $ sl (-v|--version|version)
-@sourceloop/cli/4.2.1 linux-x64 node-v18.18.2
+@sourceloop/cli/4.2.1 darwin-arm64 node-v18.17.0
```
Comment (yeshamavani) [inline packages/cli/README.md:21]: ??

File: services/audit-service/src/controllers/audit.controller.ts [hunk 2]
```diff
@@ -264,7 +264,7 @@ export class AuditController {
     let selectedAuditLogs = await this.auditLogRepository.find({
       where: where,
     });
-    if (!selectedAuditLogs) {
+    if (selectedAuditLogs.length === 0) {
```
Comment (yeshamavani) [inline services/audit-service/src/controllers/audit.controller.ts:267]: cant we if(!selectedAuditLogs.length) {}

**Review by yeshamavani — CHANGES_REQUESTED** [review]
test case for the new change?

File: services/audit-service/src/controllers/audit.controller.ts (hunk 2 above)
Comment (arpit1503khanna) [reply services/audit-service/src/controllers/audit.controller.ts:267]: yes

**Pull request merged**